actions/0074.md

```diff
@@ -1,8 +1,8 @@
 - date: 2018/01/16
-- sources: https://www.jacobinmag.com/2018/04/lanetix-tech-workers-unionization-campaign-firing, https://www.bloomberg.com/news/articles/2018-11-10/software-startup-accused-of-union-busting-will-pay-ex-employees
+- sources: https://www.jacobinmag.com/2018/04/lanetix-tech-workers-unionization-campaign-firing, https://www.bloomberg.com/news/articles/2018-11-10/software-startup-accused-of-union-busting-will-pay-ex-employees, https://gist.github.com/bwestergard/11f21ed3efa35284f653fce49178eaae, https://gist.github.com/bwestergard/11f21ed3efa35284f653fce49178eaae, https://www.sfchronicle.com/business/article/SF-tech-company-fired-software-engineers-seeking-12541301.php, https://www.bna.com/tech-startup-fires-n73014474793/, https://medium.com/tech-workers-coalition/joint-statement-of-solidarity-with-unjustly-fired-lanetix-workers-354daaa4b306, https://www.bloomberg.com/news/articles/2018-02-26/coders-want-to-unionize-with-help-from-trump, http://www.sfweekly.com/news/s-f-tech-workers-protest-for-the-right-to-unionize/, https://nplusonemag.com/issue-31/politics/code-red/, http://www.spacedaily.com/afp/180427012948.386pka56.html, https://www.youtube.com/watch?v=ItihrPELgkA#t=1h26m27s, https://www.fastcompany.com/90209362/from-amazon-to-microsoft-tech-companies-faced-yet-another-ethical-reckoning-this-week, https://www.theguardian.com/commentisfree/2018/aug/09/silicon-valley-tech-workers-labor-activism, https://www.wired.com/story/labor-board-backs-startup-engineers-fired-unionizing/, https://www.fastcompany.com/90250388/the-advocate-bjorn-westergard, https://www.fastcompany.com/90244860/silicon-valleys-new-playbook-for-tech-worker-led-resistance, https://www.vice.com/en_us/article/598kyx/the-future-of-capitalism-is-san-francisco-on-steroids-v25n3, http://www.newsguild.org/mediaguild3/software-company-learns-a-tough-lesson-about-union-busting-fired-workers-win-big-back-pay-settlement/
```

Comment:
This link isn't working for me: https://www.bna.com/tech-startup-fires-n73014474793/

This link is repeated: https://gist.github.com/bwestergard/11f21ed3efa35284f653fce49178eaae

Reply:
@nataliyaned Thanks for reviewing. I believe both problems have been fixed. Cheers!